src/main/play-doc/developer/BundleConfiguration.md

```diff
@@ -84,15 +99,28 @@ When your component will run within a container you may alternatively declare th
 ```json
     endpoints        = {
       "angular-seed-play" = {
-        protocol  = "http"
-        bind-port = 9000
-        services  = ["http://:9000"]
+        protocol      = "http"
+        bind-port     = 9000
+        service-name  = "angular-seed-play"
+        acls          = [
+          {
+            http = {
+              requests = [
+                {
+                  path-beg = "/"
+                }
+              ]
+            }
+          }
+        ]
       }
     }
 ```
 
 ### Service ports
 
+**The services is now deprecated in lieu of [request acl](RequestAclConfiguration).**
```

Comment:
As discussed in the standup, please move this part to the migration guide section.